cmd/kubeapps-apis/core/packages/v1alpha1/repositories_test.go

```diff
@@ -102,3 +132,78 @@ func TestAddPackageRepository(t *testing.T) {
 		})
 	}
 }
+
+func TestGetPackageRepositoryDetail(t *testing.T) {
+	testCases := []struct {
+		name              string
+		configuredPlugins []repoPluginsWithServer
+		statusCode        codes.Code
+		request           *corev1.GetPackageRepositoryDetailRequest
+		expectedResponse  *corev1.GetPackageRepositoryDetailResponse
+	}{
+		{
+			name: "it should successfully call the core GetPackageRepositoryDetail operation",
+			configuredPlugins: []repoPluginsWithServer{
+				mockedRepoPlugin1,
+				mockedRepoPlugin2,
+			},
+			request: &corev1.GetPackageRepositoryDetailRequest{
+				PackageRepoRef: &corev1.PackageRepositoryReference{
+					Context: &corev1.Context{
+						Cluster:   plugin_test.GlobalPackagingCluster,
+						Namespace: plugin_test.DefaultNamespace,
+					},
+					Identifier: "repo-1",
+					Plugin:     mockedPackagingPlugin1.plugin,
+				},
+			},
+			expectedResponse: &corev1.GetPackageRepositoryDetailResponse{
+				Detail: plugin_test.MakePackageRepositoryDetail("repo-1", mockedRepoPlugin1.plugin),
+			},
+			statusCode: codes.OK,
+		},
+		/*
```

Comment:
Is this commented as a reminder TODO later, or did you mean to delete it? The test mock that you modified below could return a NotFound easily enough?

Reply:
whooops. Started and forgot about it :-) Will fix